tests/console/python3_new_version_check.pm

```diff
@@ -32,27 +32,36 @@ sub run {
         add_suseconnect_product('sle-module-python3');
     }
 
-    # Test new python3 versions if any
-    my @python_versions = split(/,/, get_var('PYTHON_VERSIONS', '3.10'));
-    my $sub_version;
-    foreach my $python_version (@python_versions) {
-        record_info("python$python_version is tested now");
-        $sub_version = substr($python_version, 2);
-        my @python31x_results = split "\n", script_output("zypper se python3$sub_version | awk -F \'|\' \'/python3$sub_version/ {gsub(\" \", \"\"); print \$2}\'");
-        if ((scalar(@python31x_results) == 0) || ($python31x_results[0] ne "python3$sub_version")) {
-            die("Python $python_version not found in the repositories\n\n");
+    # Test all avaiable new python3 versions if any
+    my $ret = zypper_call('se "python3[0-9]*"', exitcode => [0, 104]);
```

Comment:
Are you sure ? https://openqa.suse.de/tests/11153979#step/python3_new_version_check/24

Reply:
[SLE15SP5_QEM](http://bassey.arch.nue2.suse.org/tests/58#step/python3_new_version_check/40)

Reply:
Now I can remove the soft failure, thanks much!